LICENSE

```diff
@@ -175,32 +175,6 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
-
-   APPENDIX: How to apply the Apache License to your work.
-
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-
 ----
 This project bundles org.slf4j:slf4j-api, org.slf4j:slf4j-log4j under the terms of the MIT license.
```

Comment:
Your distribution is not distributing slf4j, then it's not needed on the license

Reply:
Sounds good. This was part of the timeline-server-bundle which I ended up cleaning after @vinothchandar comments. Did not update the NOTICE after that. Will do. Thanks.

Reply:
@lresende : Cleaned it up. I looked at other incubator project's LICENSE files and they all have only licenses for source code contributions (part of source release). I have made the changes to be consistent with this.